cmake/ElektraCache.cmake

```diff
@@ -277,6 +277,14 @@ option (INSTALL_SYSTEM_FILES "Install files to system directories" ON)
 
 option (INSTALL_BUILD_TOOLS "Install build tools for cross-compilation" OFF)
 
+option (HASHLOOKUP "Add the possibility to lookup with an Order Preserving Minimal Perfect Hash Map" ON)
+if (HASHLOOKUP)
+	add_definitions(-DHASHLOOKUP=1)
```

Comment:
Macros should have an `ELEKTRA_` prefix. Furthermore it would be better if you only set it for the source files where needed or define it via kdbconfig.h